util/db/db.go

```diff
@@ -100,15 +100,18 @@ func NewDB(namespace string, settingsMgr *settings.SettingsManager, kubeclientse
 		ns:            namespace,
 		kubeclientset: kubeclientset,
 	}
-	clusterSecrets, err := dbInstance.listSecretsByType(common.LabelValueSecretTypeCluster)
+	dbInstance.sanityCheck()
+	return &dbInstance
+}
+
+func (db *db) sanityCheck() {
```

Comment:
I'd rename to `logInClusterWarning` - "sanity" is a bit of a loaded term. :-)

Reply:
done